data/seeds/015-recipients.js

```diff
@@ -8,21 +8,34 @@ const {
 } = require('../seedHelpers');
 
 const recipients = fakeRecipientIds.map((id) => {
+  const genID = getRand(19);
```

Comment:
In the future, these hardcoded values for each of these columns may need to be dynamically generated. (Or, ideally, we get a much better understanding of the distribution of the actual data for these values and reflect that when seeding the DB.)